srcs/executing/exec_redirs.c

```diff
@@ -42,11 +55,16 @@ static int	exec_redirs_in(t_cmd_table *cmd, int i)
 		while (1)
 		{
 			line = readline("> ");
+			if (!ft_strlen(line)) // it's ugly code but that works. Not sure how to change it.
```

Comment:
I don't understand why you need a special case for when the line is empty.

Reply:
I'm guessing it's because the rewrite function doesn't work well if you give it an empty line (which perhaps we should think if should be protected or not). But without changing anything in rewrite, I suggest to change the if statement in line 65 to if (!cmd->redirs[i].quote && ft_strlen(line))